src/scripts/main.js

```diff
@@ -354,7 +354,23 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const calculateAge = (born, died) => died - born;
+const calculateCentury = (year) => Math.ceil(year / 100);
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach((person) => {
+  const row = document.createElement('tr');
+
+  const gender = person.sex === 'm' ? 'Male' : 'Female';
```

Comment:
The gender determination logic is correct. It uses a ternary operator to assign 'Male' or 'Female' based on the `sex` property.